docs/mllib-feature-extraction.md

```diff
@@ -278,8 +278,8 @@ for details on the API.
 multiplication. In other words, it scales each column of the dataset by a scalar multiplier. This
 represents the [Hadamard product](https://en.wikipedia.org/wiki/Hadamard_product_%28matrices%29)
 between the input vector, `v` and transforming vector, `scalingVec`, to yield a result vector.
-Qu8T948*1#
```

Comment:
@jkbradley, would you mind if I ask this was mistakenly written for sure?

Reply:
@HyukjinKwon  If the original author did not reply, you can look at my latest reply and confirm this confused character sequence.

Reply:
he is an active commiter / pmc. let's wait for few more days before we go ahead.